core/runtime/dynamic_map_internal.odin

```diff
@@ -300,17 +302,36 @@ __dynamic_map_set :: proc(h: Map_Header, hash: Map_Hash, value: rawptr, loc := #
 
 	if __dynamic_map_full(h) {
 		__dynamic_map_grow(h, loc)
-		// index = __dynamic_map_find(h, hash).entry_index
-		// assert(index >= 0)
 	}
 	
 	return __dynamic_map_get_entry(h, index)
 }
 
 
+@(private="file")
+next_pow2 :: proc "contextless" (n: int) -> int {
```

Comment:
This is a bit poorly named. If you pass it a power of two, it returns that same power of two, not the next one.

e.g.
```
next_pow2(001): 1
next_pow2(002): 2
next_pow2(003): 4
next_pow2(004): 4
next_pow2(005): 8
next_pow2(006): 8
next_pow2(007): 8
next_pow2(008): 8
next_pow2(009): 16
next_pow2(010): 16
next_pow2(011): 16
next_pow2(012): 16
next_pow2(013): 16
next_pow2(014): 16
next_pow2(015): 16
next_pow2(016): 16
next_pow2(017): 32
```

Reply:
Good test!